react/components/ProductAvailabilityWrapper.tsx

```diff
@@ -116,6 +62,22 @@ function ProductAvailabilityWrapper({
 }: Props) {
   const { handles, withModifiers } = useCssHandles(CSS_HANDLES, { classes })
   const productContextValue = useProduct()
+  const intl = useIntl()
+
+  const formattedLowStockMessage = formatIOMessage({
+    id: lowStockMessage,
+    intl,
+  }) as string
+
+  const formattedHighStockMessage = formatIOMessage({
+    id: highStockMessage,
+    intl,
+  }) as string
+
+  const formattedShowAvailabilityMessage = formatIOMessage({
+    id: showAvailabilityMessage,
+    intl,
+  }) as string
```

Comment:
what do you think about memo those string values?

Reply:
Yep, good idea!